apps/trading-e2e/src/integration/capsule.cy.ts

```diff
@@ -160,44 +176,28 @@ describe('capsule', { tags: '@slow' }, () => {
     // 1001-DEPO-007
     // 1001-DEPO-008
     // 1001-DEPO-009
-    // 1002-WITH-001
-    // 1002-WITH-006
-    // 1002-WITH-009
-    // 002-WITH-011
-    // 1002-WITH-024
-    // 1002-WITH-012
-    // 1002-WITH-013
-    // 1002-WITH-014
-    // 1002-WITH-015
-    // 1002-WITH-016
-    // 1002-WITH-019
-
-    cy.visit('/#/portfolio');
-    cy.get('main[data-testid="/portfolio"]').should('exist');
-
-    cy.highlight('creating deposit');
+    // 1001-DEPO-010
 
     cy.getByTestId(depositsTab).click();
     cy.getByTestId('deposit-button').click();
     connectEthereumWallet('Unknown');
-    cy.get(assetSelectField, txTimeout).select(btcName);
+    cy.get(assetSelectField, txTimeout).select(btcName, { force: true });
```

Comment:
Is the force true needed here?

Reply:
this is necessary due to the current implementation of app: select aria-hidden="true"